docs/es6_EntityFactory.js.html

```diff
@@ -76,25 +77,31 @@ <h1 class="page-title">Source: es6/EntityFactory.js</h1>
 
 		window.D2L.Siren.EntityStore.addListener(this._href, this._token, this._onChange).then((removeListener) => {
 			this._removeListener = removeListener;
-			window.D2L.Siren.EntityStore.fetch(href, token);
+			if (entity) {
+				window.D2L.Siren.EntityStore.update(href, token, entity);
+			} else {
+				window.D2L.Siren.EntityStore.fetch(href, token);
+			}
```

Comment:
I'm trying to understand difference between update and fetch here?

Reply:
Saw your comment below, sorry i'm reading docs instead of source code

Reply:
So update takes in a siren entity that has already been fetched. So it bypasses calling fetch and inserts it into the store and notifies everyone.

The reason I am doing this is because the sequence domain has a bunch of embedded HM entities.